### ISSUE TITLE
Constant values for runtime. Tokenomics.

### ISSUE THREAD

**Issue by akorchyn** [GoldenGateGGX/golden-gate#11]
#### Description ####
We have tokenomics. Now, we are required to implement this.

#### Definition of "Done" ####
Tokenomics state is represented by [Tokenomics.md](https://github.com/GoldenGateGGX/golden-gate/blob/main/docs/tokenomics.md)

- [x] Set initial total supply [#67]
- [x] Fixed APY to 16% #47
- [x] 18 decimals #50
- [x] APY decay ladder from 16% to 2% over 30 years.  #50 
- [ ] #89
- [ ] Fixed or median commission from validator to nominators. **FOR NOW, we won't implement this** Should be able to easy implementation after #89
- [x] 10% treasury cut from validator rewards #58
- [x] 100% treasury cut from validator fees. 25% cut from tips. #47
- [x] All parameters should be configurable by OpenGov [#67]
- [ ] Slashing mechanisms. Currently, we use default Substrate parameters for slashing. Disable offline slashing.
- [x] #78 #81
- [x] #83
- [x] Disabled treasury token burning #82

**Comment by akorchyn** [comment]
https://github.com/GoldenGateGGX/golden-gate/pull/47 Added POS runtime, added static, fixed APY
https://github.com/GoldenGateGGX/golden-gate/pull/50 Added APY decay over time (once per 365.25 days). Included draft tokenomics update with the current state

**Comment by akorchyn** [comment]
#58 treasury commissions from validators reward are implemented.
